packages/rancher-logging/generated-changes/overlay/templates/loggings/k3s/logging-k3s-openrc.yaml

```diff
@@ -1,4 +1,5 @@
 {{- if and .Values.additionalLoggingSources.k3s.enabled (eq .Values.additionalLoggingSources.k3s.container_engine "openrc")}}
+{{- $total_fluentbit_tolerations := concat (.Values.tolerations) (.Values.fluentbit.tolerations) }}
```

Comment:
Put this in a helper function rather than calling it at the top of the file each time

Reply:
Fixed by avoiding the use of the variable.